package.json

```diff
@@ -43,9 +43,9 @@
   },
   "engines": {
     "node": ">=14.17.6",
-    "pnpm": ">=7.12.2 <=7.29.1"
+    "pnpm": ">=7.12.2 <=8.2.0"
   },
-  "packageManager": "pnpm@7.29.1",
```

Comment:
The pnpm version is still not match.

Reply:
My mistake, I will replace the package. json version